librz/include/rz_platform.h

```diff
@@ -13,6 +13,20 @@
 extern "C" {
 #endif
 
+#if defined(__BYTE_ORDER__)
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#define RZ_HOST_BIG_ENDIAN false
+#elif __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
+#define RZ_HOST_BIG_ENDIAN true
+#else
+#error "Unsupported endianness"
+#endif
+#else
+// Hacky version to determine endianness
+static const ut32 rz_endianness_one = 1;
+#define RZ_HOST_BIG_ENDIAN !(*((char *)&(rz_endianness_one)) == 1)
+#endif
```

Comment:
i'm quite sure this is available in `rz_types.h` as `RZ_SYS_ENDIAN` when `0` is LE, when `1` is BE